README.md

```diff
@@ -51,7 +51,17 @@ After running `docker-compose up` successfully, the output in your terminal shou
 
 - [Python 3.10](https://github.com/okfde/fragdenstaat_de/blob/21faa6893a582a02f2a96f4ccce96fddab13bec6/requirements.txt#L2) (for the backend server)
 
-> Note: You can use [pyenv](https://github.com/pyenv/pyenv) to manage your Python versions.
+    > Note: You can use [pyenv](https://github.com/pyenv/pyenv) to manage your Python versions.
```

Comment:
Can we suggest poetry instead?  I've been using poetry and the pyproject.toml to lock version numbers.

Reply:
I would prefer not to suggest either. The `pyenv` suggestion is from the `README.md` so I just copied it.

Reply:
If I don't convert everyone I know to using `pyenv`, how will the disease spread?

Reply:
I think that it's better to let people manage their software versioning by themselves. There's no easy way forward but I favor keeping things simple so that more people can happily choose to be involved in helping out.

There should be minimum versions specified and spelled out explicitly so that people are very aware of what they are.

As a compromise, it might be good to have a separate recommendations page if CodeWithAloha wants to take that position.

Reply:
Lol, I hope my joke came across as such. I'm totally okay with deleting this line and moving these types of recommendations to the Wiki, I shall leave it up to you 🚀